frontend/jupyter/cypress/e2e/form-page.cy.ts

```diff
@@ -129,7 +129,7 @@ describe('New notebook form', () => {
       // not smaller than minimum cpu
       cy.get('[data-cy-form-input="cpu"]').clear();
       cy.get('[data-cy-form-input="cpu"]').find('input').type('5.0');
-      cy.get('[data-cy-form-input="cpuLimit"]').find('input').type('4.0');
+      cy.get('[data-cy-form-input="cpuLimit"]').find('input').type('4.0', {force: true});
```

Comment:
whatcha doin?

Reply:
Ok so this one is a bit weird, following the component updates, cypress would just fail at some of these `type` commands. From what I can tell, this comes from how the new components are getting generated with the "required/mandatory" validator, as we now have the mandatory symbol "*" with our inputs. I think that although visually it doesn't appear this way, I think that this mandatory aspect is being applied OVER the actual input component. So when cypress tries to find this input, it can find it, but it thinks it's being covered by something, so it doesn't want to type what I tell it to type. So that is why I need to force it to type in the input